code/game/machinery/computer/card.dm

```diff
@@ -475,6 +482,27 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 							modify.access += access_type
 							log_id("[key_name(usr)] added [get_access_desc(access_type)] to [modify] using [scan] at [AREACOORD(usr)].")
 						playsound(src, "terminal_type", 50, 0)
+
+		if("access_give_aa")
+			if(authenticated == 2)
+				var/tried = 0
```

Comment:
you don't need this because you should be logging every attempt

Reply:
It's to prevent spamming logging in for loop once it is valid to modify the access.

Reply:
But you log stuff outside of the loop

Reply:
because logging them in the loop will `'XX gave AA to NN'*the number of accesses` to the admins. That message is enough to notice once.

Reply:
so, if it gives **anything valid** to a card, It will log. If it gives AA to AA card, It won't log.

Reply:
I changed it as `0` to `FALSE` for better recognisation.